active/TreeView/SelectionChangedEvent.md

```diff
@@ -0,0 +1,120 @@
+the reader "go read 100 pages of background information posted at ...". -->
+The `TreeView` control is lacking an event to track when selection has changed. While other list based controls (`ListView`, `ComboBox`, ...) have such API the `TreeView` control does not. 
+
+The original issue in the WinUI repo for this is here: https://github.com/microsoft/microsoft-ui-xaml/issues/322
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+This API is adding an event whenever the selected item of a `TreeView` control has changed.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+```
+private void Initialize()
+{
+    var treeView = new TreeView();
+    treeView.SelectionChanged += OnSelectionChanged;
+}
+
+private void OnSelectionChanged(object sender, TreeViewSelectionChangedEventArgs e)
+{
+    foreach (var addedItem in e.AddedItems)
+    {
+        // ...
+    }
+
+    foreach (var removedItems in e.RemovedItems)
+    {
+        // ...
+    }
+}
+```
+
+
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+The API is available as preview already in WinUI and was merged in this PR: https://github.com/microsoft/microsoft-ui-xaml/pull/2063
+
+# API Notes
+
+N/A
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+```
+runtimeclass TreeViewSelectionChangedEventArgs
+{
+    Windows.Foundation.Collections.IVector<Object> AddedItems{ get; };
+    Windows.Foundation.Collections.IVector<Object> RemovedItems{ get; };
```

Comment:
The SelectionModel class, in addition to `SelectedItems` also has `SelectedItem`. Maybe we could add that here too since we support single selection. On the other hand, we wouldn't gain much by adding that as you can also just get the item at index 0.